src/Plugin/Field/FieldFormatter/StrawberryMetadataTwigFormatter.php

```diff
@@ -186,8 +192,19 @@ public function settingsForm(array $form, FormStateInterface $form_state) {
    * {@inheritdoc}
    */
   public function settingsSummary() {
+    // Get the metadata tempalte's label for display in the summary
```

Comment:
Ups. @marlo-longley was about to merge and saw this tiny little typo. "Get the metadata tempalte's label for display in the summary" =>  template's

Reply:
@DiegoPino So. Close.
Should be fixed.

Reply:
well, well well! Should be done now. Thx.